.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        django_version: ["2.2"]
+        python_version: ["3.6", "3.7", "3.8"]
+        variant: [alpine, slim]
+        include:
+          - variant: alpine
+            pg_major: 12
+            pg_version: 12.2-r0
+          - variant: slim
+            pg_major: 10
+            pg_version: 10.13-1.pgdg100+1
+    env:
+      CI: 1
+      DOCKER_BUILDKIT: 1
```

Comment:
What is this for?

Reply:
This opts into using the newer [Docker BuildKit enhancements](https://docs.docker.com/develop/develop-images/build_enhancements/) for Docker builds. They purport some new features and performance improvements. We've been using it in [docker-tippecanoe as well](https://github.com/azavea/docker-tippecanoe/blob/master/.github/workflows/continuous-integration.yml#L18).